System/Desk/DIView.ts

```diff
@@ -148,7 +152,7 @@ export class DIView {
 
     set width(value) {
         this._width = value;
-        this.body.style.width = "".concat(value, "px");
+        this.body.style.width = "".concat(`${value}`, "px");
```

Comment:
Actually, now that I look at this again, why not `${value}px`      ?

Reply:
Yeah, that would be better

Reply:
Updated to use this pattern most of the time